lms/static/js/groups/views/cohort_discussions.js

```diff
@@ -0,0 +1,114 @@
+var edx = edx || {};
+
+(function ($, _, Backbone, gettext, interpolate_text, NotificationModel, NotificationView) {
+    'use strict';
+
+    edx.groups = edx.groups || {};
+
+    edx.groups.CohortDiscussionsView = Backbone.View.extend({
```

Comment:
I think the name ``CohortDiscussionsView`` is confusing as this isn't a view that shows discussions. Maybe it should be ``CohortDiscussionConfigurationView`` or something similar.